frontend/src/App.js

```diff
@@ -0,0 +1,65 @@
+import './App.css';
+import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom';
```

Comment:
You reference `react-router-dom` but didn't install it as a dependancy. Can you update the `package.json` when you get a chance?